website/www/site/static/js/hero/lottie-light.min.js

```diff
@@ -0,0 +1,15 @@
+(typeof navigator !== "undefined") && (function(root, factory) {
```

Comment:
This seems to be MIT licensed (based on https://github.com/airbnb/lottie-web), but it doesn't include the license and copyright as it should (for example look at our [bootstrap](https://github.com/apache/beam/blob/master/website/www/site/static/js/bootstrap.min.js). It looks like you just pulled these from https://cdnjs.com/libraries/lottie-web as recommended in airbnb.io/lottie, so maybe its not our problem? I don't know.

This PR is just merging into the website-revamp branch, so we don't need to figure it out right away, but we should figure it out before merging to master.

CC: @kennknowles wdyt?

Reply:
Yeah, the original `lottie-web` on CDN is really heavy, that's why I pulled the light version from their builds. And I followed their instruction here [1].

[1] https://github.com/airbnb/lottie-web#html

Reply:
None of the versions on there actually have a license header.
Can you add a TODO to figure out if we need to do something about this before merging to master?

Reply:
Sure!